app/templates/direct-award/view-project.html

```diff
@@ -69,141 +67,287 @@ <h1 class="govuk-heading-xl">{{ page_title }}</h1>
+      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
+    {% endif %}
+
+    <ol class="dm-task-list">
+      <li>
+        <h2 class="govuk-heading-l dm-task-list__section">
+          <span class="dm-task-list__section-number">1. </span>
+          Save a search
+        </h2>
+
+        <p class="govuk-body">
+        Save your search so you can export your results later. You must
+        <a class="govuk-link"
+           href="https://www.gov.uk/guidance/g-cloud-buyers-guide#fairness"
+           target="_blank"
+           rel="external noopener noreferrer"
+           data-analytics="trackEvent"
+           data-analytics-category="Direct Award"
+           data-analytics-action="External Link"
+         >buy fairly</a>
+        and assess all the services that you find. Do not combine the results of
+        more than one search.
+        </p>
+
+        {% if not search_summary_sentence %}
+          {{ govukButton({
+            "href": url_for("direct_award_public.choose_lot", framework_family=framework.family),
+            "text": "Start a new search",
+
+            "attributes": {
+              "data-analytics": "trackEvent",
+              "data-analytics-category": "Direct Award",
+              "data-analytics-action": "Internal Link",
+            }
+          }) }}
+
+          <p class="govuk-body">
+            <a class="govuk-link" href="{{ url_for('direct_award.saved_search_overview', framework_family=framework.family) }}">
+              See a list of your saved searches
+            </a>
+          </p>
+        {% else %}
+          <p class="app-search-summary app-search-summary--small govuk-body-s">{{ search_summary_sentence }}</p>
+
+          {% if not search.searchedAt %}
+            <p class="govuk-body">
+              <a class="govuk-link" href="{{ buyer_search_page_url }}">
+                Edit your search and view results
+              </a>
+            </p>
+          {% endif %}
+          {% if search.searchedAt or can_end_search %}
+            {{ govukTag({
+              "classes": "app-tag--box app-tag--black",
+              "text": "Completed"
+            }) }}
+          {% endif %}
+        {% endif %}
+      </li>
+
+      <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+
+      <li>
+        <h2 class="govuk-heading-l dm-task-list__section">
+          <span class="dm-task-list__section-number">2. </span>
+          Export your results
+        </h2>
+
+        <p class="govuk-body">
+          Export a list of the services you’ve found. Download suppliers’ service
+          descriptions and contact details to help you assess services. Keep a copy
+          for your records.
+        </p>
+
+        {% if project %}
+          {% if not can_end_search %}
+          <div class="govuk-form-group--error">
+            <span class="govuk-error-message">
+              <span class="govuk-visually-hidden">Error:</span>
```

Comment:
Do we need to treat this like a normal form error (ie: display an error summary and reset the page title)?

It prevents any further actions on the page, but the action to fix is to follow the previous link. Perhaps when we have an error here we could display a button/link which redirects to the search page.

Or perhaps that's not necessary until we overhaul the page.

Reply:
I think that's a good idea. We should be as consistent as we can and it's not a big change. I'll do that now.

Ideally we wouldn't let the user get to this state anyway (perhaps by greying out the save search button until they have less than 30 results, although I suppose it could be possible for the number of results to grow over time). But we'll have to leave that for later.

Reply:
Hmm, I've given it a try in 5a57dc8, what do you think? Currently the link just goes to the first item in the task list, which isn't that obvious, so maybe we should move the error message as well? Or we could just have the link in the error summary go to the search page, but that feels like surprising behaviour.

![Screenshot of error summary](https://user-images.githubusercontent.com/503614/98226646-38747b00-1f4e-11eb-885e-ca0c54d5b765.png)

Reply:
Hmm, I actually quite like moving the error message - we have a built in action, and the rest of the task list looks more consistent that way (All "cannot start yet").

Reply:
Agree we shouldn't link directly from the Error Summary

Reply:
Et voila! Commit 14517b9

<img width="1013" alt="Screenshot of error summary and error message" src="https://user-images.githubusercontent.com/503614/98229716-2bf22180-1f52-11eb-97ab-69e104004a17.png">